app/embeddings_model.py

```diff
@@ -0,0 +1,19 @@
+from sentence_transformers import SentenceTransformer
+from config.config import logger
+
+# Load the Sentence-BERT model
+try:
+    model = SentenceTransformer("sentence-transformers/all-MiniLM-L6-v2")
```

Comment:
I like to create additional abstractions from hugging face to avoid depending on a single model or library. That will make your code more flexible in case you want to change to another embedding model that might not implement the `encode` method.